.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: buildjet-8vcpu-ubuntu-2204
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: "50"
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.10.2
+
+      # Python is required because `ct lint` runs Yamale (https://github.com/23andMe/Yamale) and
+      # yamllint (https://github.com/adrienverge/yamllint) which require Python
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11.1
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.5.0
+        with:
+          version: v3.8.0
+
+      - name: Run chart-testing (lint)
+        run: ct lint --charts ./ --config=ct.yml
+                
+      - name: Create kind cluster
```

Comment:
We don't need the cluster for now as we're not testing install.

Reply:
Do you suggest we add elasticsearch + cassandra but not run the tests in CI? Also I discovered I had to upgrade the action and now it doesn't work anymore :(

This is the last build with 2.4.0 that now fails because Google deprecated some APIs
https://github.com/edmondop/helm-charts/actions/runs/9551467429/job/26325831512

using the new version fails for other reasons , see https://github.com/edmondop/helm-charts/actions/runs/9551593568/job/26326271659#step:6:21. Maybe they changed the expected directory struxture, will investigate. If you are on the Slack channel you can ping me there so we can close this

Reply:
Did ping on slack but haven't heard back. I suggested adding a new values file for the cassandra/ES case, but now that I think about it, maybe the values files doesn't matter for lint? And no, we shouldn't do the install tests yet, so don't need the Kind cluster.